docs/contracts/v3/reference/deployments/AVAX-Deployments.md

```diff
@@ -0,0 +1,59 @@
+---
+id: avax-deployments
+title: AVAX Deployments
+---
+
+# Uniswap Contract Deployments
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | AVAX Avalanch C-Chain Addresses              |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x740b1c1de25031C31FF4fC9A62f554A55cdC1baD` |
+| [Multicall](https://snowtrace.io/address/0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2/contract/43114/code)                                                     | `0x0139141Cd4Ee88dF3Cdb65881D411bAE271Ef0C2` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x9AdA7D7879214073F40183F3410F2b3f088c6381` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0xEB9fFC8bf81b4fFd11fb6A63a6B0f098c6e21950` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0x27Dd7eE7fE723e83Bf3612a75a034951fe299E99` |
```

Comment:
interesting - I dont think this one has been verified on AVAX. But I did do some checking internally and it seems like its correct. How did you originally find this address?

Reply:
same with NonfungibleTokenPositionDescriptor

Reply:
https://github.com/Uniswap/docs/pull/683#discussion_r1540120185

I think @jsy1218 grabbed the same way I have in the past, from the deployment address